rocAL/docs/examples/image_processing/decoder.py

```diff
@@ -2,11 +2,12 @@
 import sys
 from tkinter import W
```

Comment:
@rrawther Why do we need tkinter? Can you remove it?

Reply:
Done